### ISSUE TITLE
svg images not loading in Xamarin Forms 

### ISSUE THREAD

**Issue by kekecr** [luberda-molinet/FFImageLoading#1015]
### Description

I m following this course https://www.youtube.com/watch?v=WODdZ9mnvVc&t=893s

Basically, at this point I am trying to load an svg image from url. The code is exactly the same as in the samples provided, but no images are loaded when I run the app. Svg images load if I put them in the local resource folder, from uri-s I get blanks. 

I am using VS 2017, and 2.3.6. libraries of ffimagloading, I tried with 2.3.4 as in the course, but the problem was the same. 

 <ffimageloading:CachedImage WidthRequest="200" HeightRequest="200"
                Source="{Binding Land.Flag, Converter={StaticResource SvgImageSourceConverter}}">
                </ffimageloading:CachedImage>

land flag has this url in it https://restcountries.eu/data/alb.svg for example

**Comment by daniel-luberda** [comment]
Hi @kekecr, 

Make sure you: 

- called `CachedImageRenderer.Init()`
- added `SkiaSharp`, `FFImageLoading.Svg`, `FFImageLoading.Forms.Svg` packages to all your projects. 
- remove all `bin` / `obj` folders before build.

**Comment by kekecr** [comment]
Hi,

I tried with some different svg urls from the net and it worked. The one I mentioned in the above post, https://restcountries.eu/data/alb.svg, is still not working. It might be something with the svg file or something else. I don't know.

**Comment by kekecr** [comment]
I checked the output and I got this error  

System.Net.Http.HttpRequestException: An error occurred while sending the request ---> System.Net.WebException: Error: SecureChannelFailure (The authentication or decryption has failed.) ---> System.IO.IOException: The authentication or decryption has failed. ---> System.IO.IOException: The authentication or decryption has failed. ---> Mono.Security.Protocol.Tls.TlsException: The authentication or decryption has failed.04-23 17:47:19.714 I/mono-stdout(25050): FFImageLoading.DownloadAggregateException: One or more errors occurred. ---> System.Net.Http.HttpRequestException: An error occurred while sending the request ---> System.Net.WebException: Error: SecureChannelFailure (The authentication or decryption has failed.) ---> System.IO.IOException: The authentication or decryption has failed. ---> System.IO.IOException: The authentication or decryption has failed. ---> Mono.Security.Protocol.Tls.TlsException: The authentication or decry

I got it working by changing "HttpClient Implementation" setting from "Default" to "AndroidClientHandler"   and the SSL/TLS implementation to Native TLS 1.2+.  in the android project properties which can be changed at Android Project Properties -> Android Options -> Advanced
